### PR TITLE
Microserver

### PR DESCRIPTION
**Pull request by Davidde94**

### CODE REVIEW

File: Sources/Strike/Routes.swift [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+//  Created by David Evans on 22/07/2018.
+//
+
+import Foundation
+
+class Routes {
+    
+    static let AddFavourite = "/favourites/add"
+    static let RemoveFavourite = "/favourites/remove"
+    static let HunterFavoruites = "/favourites/hunterID"
```
Comment (RyanMKrol) [inline Sources/Strike/Routes.swift]: not :hunterId?

File: Sources/Strike/Microserver.swift [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+        
+        router.post(Routes.RemoveFavourite) {
+            request, response, next in
+            response.send("Favourite removed!")
+            next()
+        }
+        
+        router.get(Routes.HunterFavoruites) {
+            request, response, next in
+            let customerId: String = request.parameters["customerId"]!
```
Comment (RyanMKrol) [inline Sources/Strike/Microserver.swift]: customerId != hunterId

File: Sources/Strike/Microserver.swift [hunk 3]
```diff
@@ -0,0 +1,49 @@
+//
+//  Microserver.swift
+//  strike
+//
+//  Created by David Evans on 22/07/2018.
+//
```
Comment (RyanMKrol) [inline Sources/Strike/Microserver.swift]: Song

File: Sources/Strike/Routes.swift [hunk 4]
```diff
@@ -0,0 +1,16 @@
+//
+//  Routes.swift
+//  strike
+//
+//  Created by David Evans on 22/07/2018.
+//
```
Comment (RyanMKrol) [inline Sources/Strike/Routes.swift]: Song

File: Sources/Strike/Server.swift [hunk 5]
```diff
@@ -0,0 +1,49 @@
+//
+//  Server.swift
+//  strike
+//
+//  Created by David Evans on 22/07/2018.
+//
```
Comment (RyanMKrol) [inline Sources/Strike/Server.swift]: Song

File: Sources/Strike/Routes.swift (hunk 1 above)
Comment (Davidde94) [reply Sources/Strike/Routes.swift]: Good spot, thanks

File: Sources/Strike/Server.swift [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+        
+        router.post(Routes.RemoveFavourite) {
+            request, response, next in
+            response.send("Favourite removed!")
+            next()
+        }
+        
+        router.get(Routes.HunterFavoruites) {
+            request, response, next in
+            let customerId: String = request.parameters["customerId"]!
```
Comment (RyanMKrol) [inline Sources/Strike/Server.swift]: customerId => hunterId

File: Sources/Strike/Routes.swift [hunk 7]
```diff
@@ -0,0 +1,18 @@
...
+
+// Song: None 
+
+import Foundation
+
+class Routes {
+    
+    static let AddFavourite = "/favourites/add"
+    static let RemoveFavourite = "/favourites/remove"
+    static let HunterFavoruites = "/favourites/:hunterID"
```
Comment (RyanMKrol) [inline Sources/Strike/Routes.swift]: All of the other routes are names with actions, i.e. Add, Remove. But HunterFavourites isn't

**Pull request merged**